src/app/functions/server/getRelevantGuidedSteps.ts

```diff
@@ -70,6 +70,7 @@ export type StepLinkWithStatus = (typeof stepLinks)[number] & {
 };
 
 export function isGuidedResolutionInProgress(stepId: StepLink["id"]) {
+  if (stepId === "Scan" || stepId === "Done") return false; //this fixes a lint check
```

Comment:
This shouldn't be necessary.. The logic below is basically filtering out ^

Reply:
I know that it shouldn't be necessary, but for some reason the code wouldn't pass the lint checks.